plugins/obs-outputs/flv-mux.c

```diff
@@ -208,6 +214,26 @@ static void build_flv_meta_data(obs_output_t *context, uint8_t **output,
 	*output = bmemdup(buf, *size);
 }
 
+static void write_previous_tag_size_without_header(struct serializer *s,
+						   uint32_t header_size)
+{
+	assert(serializer_get_pos(s) >= header_size);
+	assert(serializer_get_pos(s) >= 11);
+
+	/*
+	 * From FLV file format specification version 10:
+	 * Size of previous [current] tag, including its header.
+	 * For FLV version 1 this value is 11 plus the DataSize of
+	 * the previous [current] tag.
+	 */
+	s_wb32(s, (uint32_t)serializer_get_pos(s) - header_size);
+}
+
+static void write_previous_tag_size(struct serializer *s)
```

Comment:
Should these be marked `inline`?

Reply:
I don't currently see a reason why they can't be marked as `static inline`, but I'm also not sure I see a reason why they should be marked `static inline`

let me know if you want me to change these

Reply:
I don't have a strong opinion here, so I'm fine with it as-is.

Reply:
I'd say it makes sense here, but as to whether it matters is whether it's actually a hot path or not. This is called for each tag (packet), so it's called a fair amount, but I'd say it's ultimately not going to be hot enough to make any bit of a noticeable difference to users.

But I'd be nice to have because it's a one-liner function. In terms of assembly output this is basically unnecessarily separated code at that point. Depending on compiler settings it may automatically get inlined anyway but you'd have to check the disassembly if you're *really* curious about these cases.

So I'd say might as well do it but it's not a big deal either way :woman_shrugging:

Reply:
changed to `static inline` in https://github.com/obsproject/obs-studio/pull/10494#commits-pushed-8582b2e